ethpm/package.py

```diff
@@ -215,5 +222,50 @@ def deployments(self) -> Union["Deployments", Dict[None, None]]:
             )
             for deployment_data in deployments.values()
         }
+        validate_deployments_tx_receipt(deployments, self.w3)
+        linked_deployments = get_linked_deployments(deployments)
+        if linked_deployments:
+            for deployment_data in linked_deployments.values():
+                on_chain_bytecode = self.w3.eth.getCode(
+                    to_checksum_address(deployment_data["address"])
```

Comment:
Assuming the version of web3 you're using supports it, I think it'd be better to use `to_canonical_address` since this is technically programatic access.  Neither is really better, but semantically, I think using `bytes` addresses here is slightly more appropriate.